skrifa/src/outline/autohint/style.rs

```diff
@@ -58,6 +61,31 @@ impl GlyphStyle {
             self.0 = (self.0 & !Self::STYLE_INDEX_MASK) | other.0;
         }
     }
+
+    pub(super) fn set_gsub_output(&mut self) {
```

Comment:
Naming nits: These names do not fit together so well IMO, `clear_shaping` vs. `set_gsub_output`. What does "gsub output" really mean here? Does it mean the glyph was found as corresponding to some group/class through gsub? Then maybe "set_from_gsub_substitution" + "clear_from_gsub"? But not sure if I got the semantics right.

Reply:
It means that the glyph is specifically the _output_ of a GSUB substitution. I kept the names distinct because we may want to handle GPOS as well but I agree that the disconnect is jarring with having the additional GPOS code in there to tie it together. I'll rename to `set_from_gsub_output` and `clear_from_gsub`.

Reply:
sgtm.